pontoon/base/views.py

```diff
@@ -201,9 +255,19 @@ def translate(request, locale, slug, part):
     )
 
     # Check if user authenticated
-    if not request.user.is_authenticated() and not p.pk == 1:
+    if not request.user.is_authenticated() and not project.pk == 1:
```

Comment:
I'm not sure if it's the part of this PR but i think we add new field to model which would be used instead of hardcoding id == 1.